clients/docs/confluent-cli.rst

```diff
@@ -1,7 +1,7 @@
 .. _client-examples-confluent-cli:
 
-Confluent CLI
-=============
+Apach Kafka Producer and Consumer Example: Confluent CLI
```

Comment:
```suggestion
Apache Kafka Producer and Consumer Example: Confluent CLI
```

Reply:
And this is why tokens are great, thank you.